cob_hardware_config/raw3-1/config/light.yaml

```diff
@@ -1,7 +1,9 @@
+devicedriver: ms-35
 devicestring: /dev/ttyUSB0
-baudrate: 230400
-invert_output: true
+baudrate: 38400
+num_leds: 58
+invert_output: false
```

Comment:
@ipa-bnm 
Could you please have a look at this yaml?
We used the layout from raw3-3...do you know what value to set for `devicedriver`, `baudrate` and `num_leds` for raw3-1?